code/modules/projectiles/guns/sentries.dm

```diff
@@ -49,15 +48,15 @@
 		/obj/item/ammo_magazine/sentry,
 	)
 
-/obj/item/storage/box/sentry/Initialize(mapload)
+/obj/item/storage/box/crate/sentry/Initialize(mapload)
 	. = ..()
 	new /obj/item/weapon/gun/sentry/big_sentry(src)
 	new /obj/item/ammo_magazine/sentry(src)
 
-/obj/item/storage/box/sentry/update_icon_state()
+/obj/item/storage/box/crate/sentry/update_icon_state()
 	icon_state = initial(icon_state)
 	if(!length(contents))
-		icon_state += "_e"
+		icon_state = "empty_case"
```

Comment:
why does this still exist?

Reply:
Forgot to get rid of them. I'll remove them.